subsys/net/lib/zperf/zperf_common.c

```diff
@@ -109,7 +109,7 @@ int zperf_get_ipv4_addr(char *host, struct in_addr *addr)
 	return 0;
```

Comment:
The change looks good, I just have some hard time understanding the commit message.
Perhaps change it something like:
```
When the zperf command is called with '-S' option which is means
IP_TOS for IPv4 and IPV6_TCLASS for IPv6,  an error is printed and
the setting does not work. The socket option handling was changed by
commit 77e522a5a243 ("net: context: Refactor option setters") but the
callers of option setters were not changed. This causes the IP_TOS
or IPV6_TCLASS option setting to fail. The fix is to use uint8_t to store
the value of the -S option.
```

Reply:
Hi @jukkar 
Thanks a lot, commit message updated, please review again.